stackgl_modules/index.js

```diff
@@ -13243,10 +13308,18 @@ proto.update = function(bounds, labels, labelFont, ticks, tickFont) {
       if(!ticks[d][i].text) {
         continue
       }
+
+      var font = {
+        family: ticks[d][i].font || tickFont[d].family,
```

Comment:
why does this line say `ticks[d][i]` instead of `tickFont[d]` like the line below?

Reply:
This change is reflected from https://github.com/gl-vis/gl-axes3d/pull/24/files 
`gl-axes3d` has its own API and there was a bug there which didn't pick the right item.
This is fixed now in that PR.